### ISSUE TITLE
Where are default MYSQL credentials?

### ISSUE THREAD

**Issue by sovetski** [devcontainers-contrib/features#594]
### Requested Feature

Hi,

My `devcontainer.json` is:

```json
{
  "image": "mcr.microsoft.com/devcontainers/base:ubuntu",
  "features": {
    "ghcr.io/shyim/devcontainers-features/php:0": {
	"version": 8.1
    },
    "ghcr.io/devcontainers-contrib/features/mysql-homebrew:1": {},
  },
}
```

The problem here with MYSQL, is that I don't know what are default credentials for MYSQL, to be able to connect to the database.

Thanks in advance

**Comment by vanduc2514** [comment]
I think devcontainer features in general only address the installation of additional packages / tools to the base image. So, like other devcontainer features, you can only select different version or different way of install the package. The default credential for MySQL seems like out of scope of devcontainer features. However, to achieve setting default credential of MySQL, you can try using `postCreateCommand` of devcontainer. 

```json
{
  "image": "mcr.microsoft.com/devcontainers/base:ubuntu",
  "features": {
    "ghcr.io/shyim/devcontainers-features/php:0": {
	"version": 8.1
    },
    "ghcr.io/devcontainers-contrib/features/mysql-homebrew:1": {},
  },
 "postCreateCommand": "mysqladmin -u root password 123456"
}
```  

where the user name is `root` and password is `123456`. This script will run only once, when the devcontainer is created

**Comment by evilhamsterman** [comment]
This project is not associated with the devcontainer project and appears to have been abandoned https://github.com/devcontainers-contrib/features/issues/628 . The official project has removed all devcontainer-contrib features from their listing. https://github.com/devcontainers/devcontainers.github.io/issues/451. There is a fork that is being worked on to update and maintain the features https://github.com/devcontainers-extra/features
